### PR TITLE
Create file for GSoC 2022

### PR DESCRIPTION
**Pull request by xukai92**

### CODE REVIEW

File: _posts/2022-02-17-gsoc.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+title: "Google Summer of Code 2022"
+author: Kai Xu and the TuringLang team
+date: 2022-02-17
+---
+
+It's another year of the [Google Summer of Code](https://summerofcode.withgoogle.com/) time and we've compiled an updated list of exciting Turing proects!
+Below is a list of projects the Turing team would be interested in working with students on for the summer.
+This information is also cross-posted at [Julia's Turing project page](https://julialang.org/jsoc/gsoc/turing/).
+
+If you are interested in exploring any of these projects or have any question, please reach out to the listed project mentors.
```
Comment (cpfiffer) [inline _posts/2022-02-17-gsoc.md:11]:
```suggestion
If you are interested in exploring any of these projects or have any questions, please reach out to the listed project mentors. 
```

File: _posts/2022-02-17-gsoc.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+Examples include but not limited to 
+- Forecasting ([Prophet](https://facebook.github.io/prophet/), [datasets](https://github.com/facebook/prophet/tree/main/examples))
+- Recommender system ([probabilistic matrix factorisation](http://www.cs.utoronto.ca/~amnih/papers/pmf.pdf), [dataset](https://grouplens.org/datasets/movielens/))
+- Ranking ([TrueSkill](https://en.wikipedia.org/wiki/TrueSkill), [dataset](https://github.com/dotnet/mbmlbook/tree/main/src/3.%20Meeting%20Your%20Match/Data))
+- Bayesian revenue estimation ([example](https://www.smartly.io/blog/tutorial-how-we-productized-bayesian-revenue-estimation-with-stan))
+- Political forecasting model ([example](https://github.com/sjwild/Canandian_Election_2021))
+- Topic mining (latent Dirichlet allocation and new variants)
+- Multiple Annotators/Combining Unreliable Observations (Dawid and Skene, 1979)
+
+For each model, we consider the following tasks
```
Comment (cpfiffer) [inline _posts/2022-02-17-gsoc.md:35]:
```suggestion
For each model, we would consider the following tasks as part of a GSoC project:
```

**Review by cpfiffer — APPROVED** [review]

**Comment by yebai** [comment]
thanks, @xukai92 @cpfiffer!

File: _posts/2022-02-17-gsoc.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
+---
+title: "Google Summer of Code 2022"
+author: Kai Xu and the TuringLang team
+date: 2022-02-17
+---
+
+It's another year of the [Google Summer of Code](https://summerofcode.withgoogle.com/) time and we've compiled an updated list of exciting Turing proects!
+Below is a list of projects the Turing team would be interested in working with students on for the summer.
```
Comment (trappmartin) [inline _posts/2022-02-17-gsoc.md:8]: Maybe:
Projects that the Turing team would be interested in working with students on over the summer are listed below.

**Review by trappmartin — COMMENTED** [review]
Added some minor suggestions / typo fixes

**Comment by xukai92** [comment]
I directly pushed Martin's fixes into master.

**Pull request merged**